assets/js/blocks/cart-checkout/cart/editor.scss

```diff
@@ -1,3 +1,9 @@
 .wc-block-cart__page-notice {
 	margin: 0;
 }
+
+.wp-block-woocommerce-cart.is-editor-preview {
+	max-height: 1000px;
```

Comment:
This is a hack to limit the preview height, I filled an issue in GB 
https://github.com/WordPress/gutenberg/issues/24484

Reply:
If we could display the desktop preview instead of the responsive mobile preview we would probably not have this problem, right? In addition, displaying the mobile preview in a desktop seems a bit weird.

One idea would be to extend the `EditorProvider` with an `isPreview` property. This way, inside `ContainerWidthContextProvider` we could check if it's a preview and, in that case, always apply the `isLarge` class. I don't think that should block this PR but if it's a solution that makes sense to you too, we could fill an issue.